tikv_metrics_proxy/main.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"os"
+	"os/signal"
+	"sort"
+	"strings"
+	"sync"
+	"syscall"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/juju/errors"
+	"github.com/ngaut/log"
+	"github.com/pingcap/kvproto/pkg/debugpb"
+	"github.com/pingcap/tidb-inspect-tools/pkg/utils"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/prometheus/common/expfmt"
+	"google.golang.org/grpc"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	stores = []string{}
+)
+
+type tikvOpts struct {
+	addrs string
+}
+
+// ParseHostPortAddr returns a host:port list
+func ParseHostPortAddr(s string) ([]string, error) {
```

Comment:
We have written this function earily, can u reuse it?

Reply:
I've moved it to pkg utils.